test/bats/istio-cm-patch.yaml

```diff
@@ -0,0 +1,33 @@
+data:
```

Comment:
Not sure the testing setup, but this seems to patch a lot more than what is needed. Maybe thats intended, though

Reply:
this is intended. I found it impossible to just add the `extensionProvider` to the mesh config since is a nested yaml object... it doesn't play nicely with `kubectl patch` even with a json patch. Any thoughts? Otherwise this is just to restore the rest of the default data that is created when we install istio with the demo profile for our CI